playbook/app/pb_kits/playbook/pb_form_group/_form_group.jsx

```diff
@@ -10,9 +10,22 @@ type FormGroupProps = {
   children?: Node,
   className?: string,
   data?: object,
+  fullWidth?: boolean,
   id?: string,
 }
 
+const formGroupClassName = (props: ButtonPropTypes) => {
```

Comment:
At first blush I think this could be accomplished with the `classnames` package. Maybe scope out `import classnames from 'classnames'` in other files to see what I'm talking about

Reply:
Ah it's already in the kit duh. I think you slim this down with some functions from that package.

Reply:
Nice refactor in https://github.com/powerhome/playbook/pull/1386/commits/01034374d55b4a9047ea84716c1aa779bf8c662c

Reply:
Made changes to reduce this down a lot! Thanks Cole!